src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1491,6 +1491,13 @@ internal static object GetConverter(Type type, TypeTable backupTypeTable)
                 return typesXmlConverter;
```

Comment:
@iSazonov, your last commit had 6 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Validate Attributes Tests.ValidateRange - ParameterConstuctors.Exception: ExceptionConstructingAttribute:SubstringDisambiguationEnumParseThrewAnException
```powershell
Expected strings to be the same, but they were different.
Expected length: 48
Actual length:   36
Strings differ at index 0.
Expected: 'SubstringD...'
But was:  'InvalidCastTypeConvertersConvertFrom'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/ValidateAttributes.Tests.ps1: line 86
86:                 $error[0].exception.innerexception.errorrecord.FullyQualifiedErrorId | Should -Be $InnerErrorId
```
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @{InputObject=System.Drawing.Primitives.dll; SideIndicator==>}.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Startup.Tests.ps1: line 125
125:         $diffs | Should -BeExactly $null
```
method conversion.can call all overloads of M::Foo
```powershell
Cannot convert argument "arg", with value: "10", for "Invoke" to type "System.IntPtr": "Cannot convert value "10" to type "System.IntPtr". Error: "Invalid cast from 'System.Int32' to 'System.IntPtr'.""
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Parser/Conversions.Tests.ps1: line 367
367:         $f57.Invoke(10) | Should -Be 10

```
Tests for (error, warning, etc) action preference.enum disambiguation works
```powershell
Cannot bind parameter 'ErrorAction'. Cannot convert value "Ig" to type "System.Management.Automation.ActionPreference". Error: "Ig is not a valid value for ActionPreference."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/ActionPreference.Tests.ps1: line 139
139:         Get-Process -Name asdfasdfsadfsadf -ErrorAction Ig

```
DeserializedMethods.Deserialized objects shouldn't ever have any methods (unless they are primitive known types)
```powershell
Cannot bind parameter 'MemberType'. Cannot convert value "*Method*" to type "System.Management.Automation.PSMemberTypes". Error: "*Method* is not a valid value for PSMemberTypes."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/DeserializedMethods.Tests.ps1: line 19
19:         $d | Get-Member -MemberType *Method* Add | Should -BeNullOrEmpty

```